mpas_analysis/test/test_mpas_xarray.py

```diff
@@ -2,73 +2,64 @@
 Unit test infrastructure for mpas_xarray.
 
 Xylar Asay-Davis, Phillip J. Wolfram
-12/07/2016
+02/15/2017
 """
 
 import pytest
 from mpas_analysis.test import TestCase, loaddatadir
 from mpas_analysis.shared.mpas_xarray import mpas_xarray
-import xarray as xr
 import pandas as pd
 
 
 @pytest.mark.usefixtures("loaddatadir")
-class TestNamelist(TestCase):
+class TestMpasXarray(TestCase):
 
     def test_subset_variables(self):
-        fileName = str(self.datadir.join('example_jan.nc'))
+        file_name = str(self.datadir.join('example_jan.nc'))
         timestr = ['xtime_start', 'xtime_end']
-        varList = ['time_avg_avgValueWithinOceanRegion_avgSurfaceTemperature']
+        var_list = ['time_avg_avgValueWithinOceanRegion_avgSurfaceTemperature']
 
         # first, test loading the whole data set and then calling
         # subset_variables explicitly
-        ds = xr.open_mfdataset(
-            fileName,
-            preprocess=lambda x: mpas_xarray.preprocess_mpas(x,
-                                                             timestr=timestr,
-                                                             yearoffset=1850))
-        ds = mpas_xarray.subset_variables(ds, varList)
-        self.assertEqual(sorted(ds.data_vars.keys()), sorted(varList))
+        ds = mpas_xarray.open_multifile_dataset(file_names=file_name,
+                                                time_variable_name=timestr,
+                                                year_offset=1850)
+        ds = mpas_xarray.subset_variables(ds, var_list)
```

Comment:
This was actually your test, not mine.  I just updated it.  You designed it to do the test once with preprocessing and once after the fact.  I think it's fine to do that, though I also think it's unlikely to break.